packages/office-ui-fabric-react/src/components/TeachingBubble/TeachingBubble.styles.ts

```diff
@@ -197,9 +197,10 @@ export const getStyles = (props: ITeachingBubbleStyleProps): ITeachingBubbleStyl
       classNames.footer,
       {
         display: 'flex',
+        alignItems: 'center',
+        color: palette.white,
```

Comment:
A couple of questions:
* Will the color affect styling of the buttons?
* Isn't the alignment a styling change? Content seems to be left-aligned, currently.

Reply:
The colors for the buttons are defined on lines 255 and 283 with more specific selectors. https://github.com/OfficeDev/office-ui-fabric-react/blob/45baec2586a2ad0232fc0c6c76864b66a4730fbc/packages/office-ui-fabric-react/src/components/TeachingBubble/TeachingBubble.styles.ts#L255


I don't think the alignment will change anything in effect -- this affects vertical alignment only, and the containing div's height is not defined so there would be no visible change. The center alignment will make the custom footer text center-aligned vertically, and if the text is long enough, then the buttons would be vertically centered as well within the footer container. Let me know if my thinking is correct.

Reply:
This would be a great place to have the teachingbubble be a 'strong' theme, so that a Text component would automatically be white.